src/exclusive-or.rst

```diff
@@ -77,21 +77,20 @@ These rules also imply :math:`a \xor b \xor a = b`:
                    & = b               & \; & \text{(fourth rule)}
    \end{aligned}
 
-We'll use this property often when using XOR for encryption; you can
-think of that first XOR with :math:`a` as encrypting, and the second one
-as decrypting.
+We use this property for XOR encryption. The first XOR :math:`a` can be thought of 
+as encrypting, and the second one as decrypting.
 
 Bitwise XOR
 ~~~~~~~~~~~
 
-XOR, as we've just defined it, operates only on single bits or Boolean
-values. Since we usually deal with values comprised of many bits, most
-programming languages provide a “bitwise XOR” operator: an operator that
+XOR, as defined, only operates on single bits or Boolean
```

Comment:
"as defined" is a little tricky. XOR is a binary operation so doesn't have to be single bits or Boolean values.